app/pages/help.html

```diff
@@ -131,6 +131,8 @@ <h3 id=":reload">:reload</h3>
             <li>Finally run the <span class="command-block">:reload</span> command to run all viebrc commands</li>
         </ul>
         Of course it's also possible to make a mapping or custom command to do this for you.
+        <h3 id=":restart">:restart</h3>
+        This command will restart Vieb.  It is functionally equivalent to running the <a href="#:quitall">:quitall</a> command, and then starting Vieb again.
```

Comment:
I think the example section should be present for all commands, even if it's simple, because it shows just how simple the command is in a short overview: "`:restart` - Restart Vieb". I like the current description, no need for changes there.